ingestion/src/metadata/profiler/orm/converter.py

```diff
@@ -104,6 +104,19 @@ def check_snowflake_case_sensitive(table_service_type, table_or_col) -> Optional
 
     return None
 
+def check_if_should_quote_column_name(table_service_type) -> Optional[bool]:
+    """Check whether column name should be quoted when passed into the sql command build up.
+    This is important when a column name is the same as a reserve word and causes a sql error.
+    
+    Args:
+        table_service_type: the main sql engine to determine if we should always quote.
+    Return: True or False
+    """
+    
+    if table_service_type == databaseService.DatabaseServiceType.Hive:
+        return True
+    
+    return None
```

Comment:
@ksirmons do you think we should add this logic to `check_snowflake_case_sensitive` and renamed `check_snowflake_case_sensitive` to `quote_names`?

Reply:
Actually I think we should always quote the column names no matter the db engine,  unless there is a place to check the column name against a reserve word list.

Reply:
@ksirmons sqlalchemy will automatically quote the names based on the engine backend implementation if this argument is set to `None` -- which is the default value. In certain scenarios (i.e. snowflake case sensitive column names) we had to override it and set it to True.